lib/plain/endpoints/environment-alias.ts

```diff
@@ -0,0 +1,69 @@
+import { AxiosInstance } from 'axios'
+import * as raw from './raw'
+import cloneDeep from 'lodash/cloneDeep'
+import {
+  EnvironmentAliasProps,
+  CreateEnvironmentAliasProps,
+} from '../../entities/environment-alias'
+import { CollectionProp, PaginationQueryParams, GetSpaceParams } from './common-types'
+
+type GetSpaceEnvAliasParams = GetSpaceParams & { environmentAliasId: string }
+
```

Comment:
I like the `getBaseUrl` pattern we started to introduce and would love to see it in here too.